package.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "react-prebid",
-    "version": "2.1.1",
+    "name": "react-advertising",
+    "version": "3.0.0-beta.5",
     "description": "Library for ad placements with Prebid header bidding in React applications",
     "main": "lib/index.js",
-    "unpkg": "dist/index.min.js",
+    "unpkg": "dist/react-prebid.min.js",
```

Comment:
If you want to change library name to `react-advertising` then we could update `prebid` name usage as well?

Reply:
Yes, absolutely, thanks for pointing it out. This PR is work in progress.